mslib/msui/kmloverlay_dockwidget.py

```diff
@@ -32,9 +32,10 @@
 import os
 from matplotlib import patheffects
 
-from mslib.msui.mss_qt import QtGui, QtWidgets, QtCore, get_open_filenames, get_save_filename
+from mslib.msui.mss_qt import get_open_filenames, get_save_filename
 from mslib.msui.mss_qt import ui_kmloverlay_dockwidget as ui
 from mslib.msui.mss_qt import ui_customize_kml
+from PyQt5 import QtGui, QtWidgets, QtCore
```

Comment:
Stylistically imports should be ordered from standard libraries/foreign libraries/our library

Reply:
@joernu76 I could run Pycharms "Optimize Imports" on the MSS project. 
This will automatically group them by standard/foreign/local libraries, which seems to have been done incorrectly in a lot of files actually.

Is this something you want me to do?

Reply:
Yes and no. Doing this manually sucks (and I normally only ask to observe this when doing changes oneself, not fixing it everywhere where it isn't the case), but I have yet to find a tool that doesn't sometime cause more trouble than its worth.
Certainly do not use this tool here. The order of imports is also not that important that I wouldn't merge it (there are some special cases where one cannot follow this rule, for which reason I also disable the corresponding flake8 check often).

Also autoformatting is something that works only 95% of the time, even though I'd like to like, e.g. 'black'.

But you could prepare a separate pull request, where we could review the proposed changes. I haven't used PyCharmed in two years, so they might have gotten smarter... Largely it should be fine, currently, but a tool will probably do a LOT of resorting.

Reply:
The cleanup of order for imports on other modules should be an refactoring issue. It is not so important yet. I would prefer a better coverage first, 85 - 90%.

Reply:
Obviously, it is rather unimportant, but probably *much* easier to do (if the PyCharmed tool does it).